coriolis/api/v1/replicas.py

```diff
@@ -91,11 +94,19 @@ def create(self, req, body):
             raise exc.HTTPBadRequest(
                 explanation="Invalid destination "
                             "environment: %s" % message)
+        is_valid, message = (
```

Comment:
Please move this check in `_validate_create_body()` to be consistent with the dest-env validation.